internal/controller/standalone_pgadmin/pod.go

```diff
@@ -25,6 +26,10 @@ import (
 	"github.com/crunchydata/postgres-operator/pkg/apis/postgres-operator.crunchydata.com/v1beta1"
 )
 
+const (
+	configMountPath = "/etc/pgadmin"
```

Comment:
💭 IMV, this path is dangerous because pgAdmin executes Python from here. I sense that I want more commentary about how this path/constant is different from similarly named ones in other packages. Like, it isn't _the_ one directory where we plop files. Perhaps as more constants appear, the purpose of this path will be more clear and I'll feel better.